.devcontainer/devcontainer.json

```diff
@@ -1,31 +1,52 @@
 {
-	"name": "Go",
-	"build": {
-		"dockerfile": "Dockerfile",
-		// Update the VARIANT arg to pick a version of Go
-		"args": { "VARIANT": "1" }
-	},
-	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+  "name": "Go",
+  "build": {
+    "dockerfile": "Dockerfile",
+    // Update the VARIANT arg to pick a version of Go
+    "args": { "VARIANT": "1.14" }
+  },
```

Comment:
Leave this as it is - 1.15 release is pretty close.
I think setting 1 will pick up the container with the latest stable go.

Reply:
Yes, you're right.